glib/src/win32.rs

```diff
@@ -0,0 +1,18 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::translate::*;
+use std::ffi::OsString;
+
+#[doc(alias = "g_win32_get_package_installation_directory_of_module")]
+pub fn win32_get_package_installation_directory_of_module(
+    hmodule: ffi::gpointer,
+) -> Option<OsString> {
```

Comment:
Shall I return `PathBuf` instead of `OsString`?

Reply:
`PathBuf` if it's actually a path :)